app.js

```diff
@@ -208,17 +210,6 @@ app.runOnStartup = function() {
 	var rpcCredentials = null;
 	if (config.credentials.rpc) {
 		rpcCredentials = config.credentials.rpc;
-
```

Comment:
I changed this in a slightly different way. I think it is cleaner.
Check it out:
https://github.com/janoside/btc-rpc-explorer/pull/56/files#diff-0364f57fbff2fabbe941ed20c328ef1aL69

Reply:
You're right, `config.credentials.rpc` could never actually be empty, the condition checking for this is unnecessary. Will update in a moment.